source/css/scss/app/pages/finding.scss

```diff
@@ -116,6 +116,19 @@ $filter-open-behavior-query: "#{$screen} and (min-width: #{$filter-open-behavior
 	}
 }
 
+.heading-region {
+  	margin-top: rem-calc(20);
+  	padding: 0 $loan-card-list-margin/2;
+
+  	@media #{$large-up} {
+		padding: 0 $loan-card-list-margin;
+  	}
+
+  	@media (min-width:1052px) {
```

Comment:
@media #{$filter-open-behavior-query}

Reply:
FYI: this suggested media query didn't work as planned. Went with: @media #{$gutter-up-query}